src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/delete_interaction_choice_set_request.h

```diff
@@ -43,6 +46,8 @@ namespace app_mngr = application_manager;
 
 namespace commands {
 
+typedef std::set<std::uint32_t> SentRequestsSet;
```

Comment:
@IGapchuk you don't need to specify std for this type

Reply:
@BSolonenko Deleted in 05a0ac563f1b887f483b99518f63582d86643583.